cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -1151,6 +1403,37 @@ fn x64_get_regs(inst: &Inst, collector: &mut RegUsageCollector) {
             src.get_regs_as_uses(collector);
             collector.add_def(*dst);
         }
+        Inst::CvtUint64ToFloatSeq {
+            src,
+            dst,
+            tmp_gpr1,
+            tmp_gpr2,
+            ..
+        } => {
+            collector.add_use(*src);
+            collector.add_def(*dst);
+            collector.add_def(*tmp_gpr1);
+            collector.add_def(*tmp_gpr2);
```

Comment:
See also my comment above about need for clear analysis/policy/audit of temp register uses.

Reply:
Good catch for `CvtUint64ToFloatSeq`, scary that testing didn't stumble upon it. Agreed we'll need a better policy in this kind of situation; in the meantime, I'll mark it as `mod` instead of `use`, and will make sure to add a copy of the real src to a temporary fed to this Inst's src; this is sufficient to ensure the src and tmps will get different registers (otherwise, they'd be conflicting writes). Thanks!

Other sequences use `mod` for the src operand for this same reason.

[and several minutes later] Even scarier, the checked div or rem sequence had the same potential bug. Changed it to mark the `divisor` operand as modified too.